config/application.properties

```diff
@@ -31,4 +31,10 @@ http.port=8080
 #Enable CORS
 http.cors.enabled=false
 http.cors.allowedOrigins=*
-http.cors.allowedMethods=GET,POST,PUT,DELETE,OPTIONS,PATCH
\ No newline at end of file
+http.cors.allowedMethods=GET,POST,PUT,DELETE,OPTIONS,PATCH
+
+#Disable consumer
+http.consumer.enabled=false
+
+#Disable producer
+http.producer.enabled=true
```

Comment:
As I said before -> I think this should be enabled by default as that is the original state.

Reply:
Agree. Both producer and consumer has to be enabled by default.